### PR TITLE
contrib/lazydocker: new package (0.23.3)

### PR DESCRIPTION
**Pull request by ttyyls**

### CODE REVIEW

File: contrib/lazydocker/template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+pkgname = "lazydocker"
+pkgver = "0.23.3"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Lazier way to manage docker"
```
Comment (q66) [inline contrib/lazydocker/template.py:6]: that's an undescriptive pkgdesc if i've ever seen one

Comment (ttyyls) [reply contrib/lazydocker/template.py:6]: that's just the repo description though...

Comment (q66) [reply contrib/lazydocker/template.py:6]: there is no rule or precedent saying pkgdesc should be the same as upstream repo description

Comment (ttyyls) [reply contrib/lazydocker/template.py:6]: i wouldn't go so far as to argue from precedants but fwiw a pkgdesc acts simply as a tertiary data point to quickly confirm wether the package one is inspecting from the package manager is the same as the one in mind alongside the name, and version with the url pointing to the homepage if available or docs or upstream repo.

All to say that pkgdesc that get too creative might harm that purpose as i understand it at least, but i digress.

File: contrib/lazydocker/template.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+pkgname = "lazydocker"
+pkgver = "0.23.3"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "TUI for docker resource visualization and management"
+maintainer = "ttyyls <contact@behri.org>"
+license = "MIT"
+url = "https://github.com/jesseduffield/lazydocker"
+source = f"{url}/archive/refs/tags/v0.23.3.tar.gz"
```
Comment (nekopsykose) [inline contrib/lazydocker/template.py:10]:
```suggestion
source = f"{url}/archive/refs/tags/v{pkgver}.tar.gz"
```

**Pull request merged**